packages/inter-protocol/src/reserve/assetReserveKit.js

```diff
@@ -166,6 +198,67 @@ export const prepareAssetReserveKit = async (
           facets.helper.writeMetrics();
         },
       },
+      withdrawalFacet: {
+        Withdraw() {
+          /** @type {OfferHandler<Promise<string>>} */
+          const handler = async seat => {
+            try {
+              guard.assert();
+              const { state } = this;
```

Comment:
I had to carefully audit the code in this method to determine that `state` is not modified. To make it obvious at a glance, I suggest:

```suggestion
              const { collateralSeat } = this.state;
```

Reply:
Done, thanks!